src/core/RandomVariables.jl

```diff
@@ -1169,6 +1215,30 @@ function show(io::IO, vi::UntypedVarInfo)
     print(io, vi_str)
 end
 
+function show(io::IO, vi::TypedVarInfo)
```

Comment:
Why the type is strict to `TypedVarInfo`?

Reply:
Because they have different structures -- `UntypedVarInfo` has explicit field names, whereas a `TypedVarInfo` splits all the information into various parts of a `NamedTuple`.